<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MieteDlg</class>
 <widget class="QDialog" name="MieteDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>465</width>
    <height>360</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>360</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>465</width>
    <height>360</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Mietanpassung</string>
  </property>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>284</x>
     <y>70</y>
     <width>175</width>
     <height>250</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>175</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="title">
    <string>Folgemiete</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="0" column="0">
     <widget class="QLineEdit" name="inNettoMiete_neu">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLineEdit" name="inMieteGueltigAb_neu">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLineEdit" name="inNkAbschlag_neu">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLineEdit" name="inMieteGueltigBis_neu">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QToolButton" name="tbMieteGueltigAb_neu">
      <property name="text">
       <string>...</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QToolButton" name="tbMieteGueltigBis_neu">
      <property name="text">
       <string>...</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QToolButton" name="tbResetMietGueltigAb_neu">
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>icons/x.png</normaloff>icons/x.png</iconset>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QToolButton" name="tbResetMietGueltigBis_neu">
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>icons/x.png</normaloff>icons/x.png</iconset>
      </property>
     </widget>
    </item>
    <item row="4" column="0" colspan="3">
     <widget class="QPlainTextEdit" name="txtMieteBemerk_neu">
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>19</x>
     <y>326</y>
     <width>189</width>
     <height>25</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>105</x>
     <y>70</y>
     <width>175</width>
     <height>250</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>175</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="title">
    <string>Miete</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QLineEdit" name="inNettoMiete">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="inputMask">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLineEdit" name="inMieteGueltigBis">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>1</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLineEdit" name="inMieteGueltigAb">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLineEdit" name="inNkAbschlag">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QToolButton" name="tbMieteGueltigAb">
      <property name="text">
       <string>...</string>
      </property>
      <property name="iconSize">
       <size>
        <width>18</width>
        <height>18</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QToolButton" name="tbMieteGueltigBis">
      <property name="text">
       <string>...</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QToolButton" name="tbResetMietGueltigAb">
      <property name="minimumSize">
       <size>
        <width>26</width>
        <height>24</height>
       </size>
      </property>
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>icons/x.png</normaloff>icons/x.png</iconset>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QToolButton" name="tbResetMietGueltigBis">
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>icons/x.png</normaloff>icons/x.png</iconset>
      </property>
     </widget>
    </item>
    <item row="4" column="0" colspan="3">
     <widget class="QPlainTextEdit" name="txtMieteBemerk">
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="plainText">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="lblWohnung">
   <property name="geometry">
    <rect>
     <x>8</x>
     <y>9</y>
     <width>451</width>
     <height>55</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>55</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>55</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(186, 189, 182);</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="textFormat">
    <enum>Qt::RichText</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>6</x>
     <y>100</y>
     <width>101</width>
     <height>30</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>30</height>
    </size>
   </property>
   <property name="text">
    <string>Netto-Miete:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>8</x>
     <y>130</y>
     <width>101</width>
     <height>30</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>30</height>
    </size>
   </property>
   <property name="text">
    <string>NK-Abschlag:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>8</x>
     <y>192</y>
     <width>101</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>30</height>
    </size>
   </property>
   <property name="text">
    <string>Gültig bis:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>8</x>
     <y>231</y>
     <width>101</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Bemerkung:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>6</x>
     <y>161</y>
     <width>101</width>
     <height>30</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>30</height>
    </size>
   </property>
   <property name="text">
    <string>Gültig ab:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>inNettoMiete_neu</tabstop>
  <tabstop>inNkAbschlag_neu</tabstop>
  <tabstop>inMieteGueltigAb_neu</tabstop>
  <tabstop>tbMieteGueltigAb_neu</tabstop>
  <tabstop>inMieteGueltigBis_neu</tabstop>
  <tabstop>tbMieteGueltigBis_neu</tabstop>
  <tabstop>txtMieteBemerk_neu</tabstop>
  <tabstop>inNettoMiete</tabstop>
  <tabstop>inNkAbschlag</tabstop>
  <tabstop>inMieteGueltigAb</tabstop>
  <tabstop>tbMieteGueltigAb</tabstop>
  <tabstop>inMieteGueltigBis</tabstop>
  <tabstop>tbMieteGueltigBis</tabstop>
  <tabstop>txtMieteBemerk</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>MieteDlg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>MieteDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tbMieteGueltigAb</sender>
   <signal>clicked(bool)</signal>
   <receiver>MieteDlg</receiver>
   <slot>onMieteGueltigAbClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>192</x>
     <y>181</y>
    </hint>
    <hint type="destinationlabel">
     <x>11</x>
     <y>267</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tbMieteGueltigBis</sender>
   <signal>clicked()</signal>
   <receiver>MieteDlg</receiver>
   <slot>onMieteGueltigBisClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>198</x>
     <y>210</y>
    </hint>
    <hint type="destinationlabel">
     <x>3</x>
     <y>226</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tbMieteGueltigAb_neu</sender>
   <signal>clicked()</signal>
   <receiver>MieteDlg</receiver>
   <slot>onMieteGueltigAbClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>358</x>
     <y>178</y>
    </hint>
    <hint type="destinationlabel">
     <x>412</x>
     <y>249</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tbMieteGueltigBis_neu</sender>
   <signal>clicked()</signal>
   <receiver>MieteDlg</receiver>
   <slot>onMieteGueltigBisClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>351</x>
     <y>212</y>
    </hint>
    <hint type="destinationlabel">
     <x>416</x>
     <y>216</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tbResetMietGueltigAb</sender>
   <signal>clicked()</signal>
   <receiver>MieteDlg</receiver>
   <slot>onDatumReset()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>236</x>
     <y>172</y>
    </hint>
    <hint type="destinationlabel">
     <x>412</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tbResetMietGueltigBis</sender>
   <signal>clicked()</signal>
   <receiver>MieteDlg</receiver>
   <slot>onDatumReset()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>227</x>
     <y>209</y>
    </hint>
    <hint type="destinationlabel">
     <x>415</x>
     <y>188</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tbResetMietGueltigAb_neu</sender>
   <signal>clicked()</signal>
   <receiver>MieteDlg</receiver>
   <slot>onDatumReset()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>387</x>
     <y>176</y>
    </hint>
    <hint type="destinationlabel">
     <x>416</x>
     <y>176</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tbResetMietGueltigBis_neu</sender>
   <signal>clicked()</signal>
   <receiver>MieteDlg</receiver>
   <slot>onDatumReset()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>388</x>
     <y>206</y>
    </hint>
    <hint type="destinationlabel">
     <x>415</x>
     <y>288</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onMieteGueltigAbClicked()</slot>
  <slot>onMieteGueltigBisClicked()</slot>
  <slot>onMieteGueltigAbNeuClicked(senderName)</slot>
  <slot>onDatumReset()</slot>
 </slots>
</ui>
